doc/awx-setup.md

```diff
@@ -0,0 +1,46 @@
+# Recreating the AWX server
+
+First step is to run the AWX playbook:
+
+```sh 
+$ ansible-playbook playbooks/create-awx.yml
+```
+
+_note_: the playbook is currently targeted against `infra-ibm-ubuntu2004-x64-1` so if that changes don't forget to update the playbook
+
+# Manual Setup
+
+## Authentication
+
+To setup the authentication for AWX you need the github OAuth credentials which can be found in the secrets repo (Infra level).
```

Comment:
```suggestion
To setup the authentication for AWX you need the GitHub OAuth credentials which can be found in the secrets repo (Infra level).
```

Reply:
I don't think the OAuth credentials have been added to secrets yet?

Reply:
nope, they will be I dont have the required access so will need to ask a member of infra to add them.